src/components/Search/SearchOverTime.tsx

```diff
@@ -82,12 +81,23 @@ export default function SearchOverTime({
     },
   });
 
+  const timeRangeText = timeRanges.find(
+    (entry) => entry.value === timeRangeValue,
+  )?.text;
+
+  const readOnlyStyles = style({
+    borderRadius: Spacing.xSmall,
+    backgroundColor: Colors.Background200,
+    padding: `${Spacing.xSmall}px ${Spacing.xSmall}px`,
+  });
+
   return (
     <Grid className={styles.component}>
       {/**** Base - Time-Range Labels ****/}
       <Grid
         container
         spacing={2}
+        mb={0.5}
```

Comment:
This added margin looks wrong. Indeed on my computer it seems to be too much, and especially it seems be counted in addition to the other margins, and therefore the visual margin below "Base repository" is wider than the one Below "Revisions".

Reply:
I'll do a styles cleanup PR next.